src/chat/zulip/triagebot.md

```diff
@@ -0,0 +1,220 @@
+# Triagebot
+
+Triaging on the rust-lang repository is an important step to prepare the pipeline for the rust-lang teams that will take care of issues and regressions. The triagebot is the assistant for the rust-lang organization.
```

Comment:
I thought to give a bit of context, how I improve on this?

Reply:
I think we should focus on the fact that Triagebot is there to allow *anyone* to help, by assigning themselves, or labeling issues, to help with triage without being a member of the organisation.